package-lock.json

```diff
@@ -0,0 +1,5433 @@
+{
```

Comment:
Never commit `package-lock.json`. We use [bun](https://bun.sh/) in Capgo, and while I personally use NPM when doing local development, it is unacceptable to commit `package-lock.json`